File: _wpf/wpf-business-programmer/3-wpf-business-programmer-m3-exercise-files/Samples-2012/VB/WPFTemplateApp/WPFTemplateApp/Common/WPFCommon.vb
Public Class WPFCommon
  Public Shared Function GetCurrentDirectory() As String
    Dim path As String = Nothing

    path = AppDomain.CurrentDomain.BaseDirectory
    If path.IndexOf("\bin") > 0 Then
      path = path.Substring(0, path.LastIndexOf("\bin"))
    End If

    Return path
  End Function
End Class

File: _wpf/wpf-business-programmer/3-wpf-business-programmer-m3-exercise-files/Samples/CSharp/WPFTemplateApp/WPFTemplateApp/Common/WPFCommon.cs
using System;

namespace WPFTemplateApp
{
  public class WPFCommon
  {
    public static string GetCurrentDirectory()
    {
      string path = null;

      path = AppDomain.CurrentDomain.BaseDirectory;
      if (path.IndexOf(@"\bin") > 0)
      {
        path = path.Substring(0, path.LastIndexOf(@"\bin"));
      }

      return path;
    }
  }
}
